docs/concepts/macros/sqlmesh_macros.md

```diff
@@ -911,6 +911,37 @@ The methods are available because the `column` argument is parsed as a SQLGlot [
 
 Column expressions are sub-classes of the [Condition class](https://sqlglot.com/sqlglot/expressions.html#Condition), so they have builder methods like [`between`](https://sqlglot.com/sqlglot/expressions.html#Condition.between) and [`like`](https://sqlglot.com/sqlglot/expressions.html#Condition.like).
 
+#### Accessing model schemas
+
+Model schemas can be accessed within a Python macro function through its evaluation context's `column_to_types` method, when they can be statically determined.
+
+For example, consider the following macro function which aims to rename the columns of a target model by adding a prefix to them:
+
+```python linenums="1"
+from sqlglot import exp
+from sqlmesh.core.macros import macro
+
+@macro()
+def foo(evaluator, model_name, prefix):
+    prefix = prefix.name
+    renamed_projections = []
+
+    for name, dtype in evaluator.columns_to_types(model_name.sql()).items():
```

Comment:
Pull out `model_name.sql()` into its own line and add comment describing what it does

Reply:
Good point, thanks!